constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt

```diff
@@ -484,14 +484,19 @@ interface Dimension {
 
     companion object {
         /**
-         * Creates a [Dimension] representing a suggested dp size. The requested size will
-         * be respected unless the constraints in the [ConstraintSet] do not allow it. The min
-         * and max bounds will be respected regardless of the constraints in the [ConstraintSet].
+         * Links should be specified from both sides corresponding to this dimension, in order for
+         * this to work.
+         *
+         * Creates a [Dimension] such that if the constraints allow it, will have the size given by
+         * [dp], otherwise will take the size remaining within the constraints.
+         *
+         * This is effectively a shorthand for [fillToConstraints] with a max value.
+         *
          * To make the value fixed (respected regardless the [ConstraintSet]), [value] should
          * be used instead.
          */
-        fun preferredValue(dp: Dp): Dimension.Coercible =
-            DimensionDescription { state -> SolverDimension.Suggested(state.convertDimension(dp)) }
+        fun preferredValue(dp: Dp): Dimension.MinCoercible =
```

Comment:
Only allow to change the min value, since the max is getting set with the given value.